tools/c7n_azure/c7n_azure/filters.py

```diff
@@ -28,27 +27,15 @@ class MetricFilter(Filter):
         'total': Math.sum
     }
 
-    ops = {
-        'eq': operator.eq,
-        'equal': operator.eq,
-        'ne': operator.ne,
-        'not-equal': operator.ne,
-        'gt': operator.gt,
-        'greater-than': operator.gt,
-        'ge': operator.ge,
-        'gte': operator.ge,
-        'le': operator.le,
-        'lte': operator.le,
-        'lt': operator.lt,
-        'less-than': operator.lt
-    }
-
     schema = {
         'type': 'object',
         'required': ['type', 'metric', 'op', 'threshold'],
         'properties': {
             'metric': {'type': 'string'},
-            'op': {'enum': list(ops.keys())},
+            'op': {'enum': [
+                'eq', 'equal', 'ne', 'not-equal', 'gt', 'greater-than', 'ge',
+                'gte', 'le', 'lte', 'lt', 'less-than'
+            ]},
```

Comment:
Seems worse?  `'op': {'enum': list(OPERATORS.keys())}}}` 

Were you having problems with docs on it? or?

Reply:
That was the original approach, but some of the keys in `OPERATORS` didn't seem to make a whole lot of sense.

Reply:
And I switched it from using an in-class `operators` to `OPERATORS` as defined in `c7n.filters.core`

Reply:
Oh! I see now.  I'm not sure I see it as better than what you had though?  Decoupling the schema from the list there might make maintaining it harder if that source list changes, etc.  I'd probably leave it like you had it before, or use the new list and let all the operators in, unless there is some other benefit I missed beyond avoiding the semi-duplicate operator map (not a problem I think).

Reply:
👍 I'll switch it back to how it was before